<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Preferences</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#listViewPersonal, #listViewSystem, #listViewHardware {
	border: none;
	background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageList">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupPersonal">
          <property name="title">
           <string>Personal</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListView" name="listViewPersonal">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="autoScroll">
              <bool>false</bool>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="movement">
              <enum>QListView::Snap</enum>
             </property>
             <property name="viewMode">
              <enum>QListView::IconMode</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxHardware">
          <property name="title">
           <string>Hardware</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListView" name="listViewHardware">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="autoScroll">
              <bool>false</bool>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="movement">
              <enum>QListView::Snap</enum>
             </property>
             <property name="viewMode">
              <enum>QListView::IconMode</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxSystem">
          <property name="title">
           <string>System</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QListView" name="listViewSystem">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="autoScroll">
              <bool>false</bool>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="movement">
              <enum>QListView::Snap</enum>
             </property>
             <property name="viewMode">
              <enum>QListView::IconMode</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowAll"/>
  </widget>
  <action name="actionShowAll">
   <property name="text">
    <string>Show All</string>
   </property>
   <property name="toolTip">
    <string>Return to the list of preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
